pkg/urbit/jets/d/in_uni.c

```diff
@@ -5,100 +5,93 @@
 
 /* internal functions
 */
-
 static u3_noun
 _in_uni(u3_noun a, u3_noun b)
 {
-  u3_noun n_a, l_a, r_a,
-          n_b, l_b, r_b,
-          neb, sub, naw, pro;
   if ( u3_nul == a ) {
     return u3k(b);
   }
   else if ( u3_nul == b ) {
     return u3k(a);
   }
-  else if ( (c3n == u3r_trel(a, &n_a, &l_a, &r_a))
-         || (c3n == u3r_trel(b, &n_b, &l_b, &r_b)) )
-  {
-    return u3m_bail(c3__exit);
-  }
-  else if ( c3n == u3qc_mor(n_a, n_b) ) {
-    if ( c3y == u3r_sing(n_a, n_b) ) {
+  else {
+    u3_noun n_a, l_a, r_a,
+            n_b, l_b, r_b,
+            neb, sub, naw, pro;
+    u3x_trel(a, &n_a, &l_a, &r_a);
+    u3r_trel(b, &n_b, &l_b, &r_b);
```

Comment:
Again, I'm not sure why the second one here is a u3r instead of a u3x.  Maybe there's a good reason for it after all, although if so it's not jumping out at me.

Reply:
same here